src/crate/theme/rtd/crate/sections/sidebar-secondary.html

```diff
@@ -10,7 +10,7 @@
       </script>
       {%- endif %}
     
-      {%- if display_version %}
+      {%- if versions %}
```

Comment:
We are discussing this over here.

- https://github.com/crate/crate-docs-theme/issues/517#issuecomment-2275931317

Reply:
When adjusting into that direction, the version chooser may re-appear on other projects where we wanted to hide it? Shall we just bake a new devX release including your updates, in order to just try it out?